### ISSUE TITLE
Explore accessibility of drag-n-drop interface

### ISSUE THREAD

**Issue by awead** [psu-libraries/cho#295]
Relates #250 and #251. Decisions about UI elements will impact the implementation of the above tickets.

**Comment by awead** [comment]
Leading candidates are:
1. https://github.com/salesforce-ux/dnd-a11y-patterns
With an accessible ordering: https://salesforce-ux.github.io/dnd-a11y-patterns/#/sortB?_k=tpa21c
And a drag-n-drop that's partcially accessible: https://salesforce-ux.github.io/dnd-a11y-patterns/#/bucket?_k=n5rkaq
Note that the ordering is not accessible as in the first example.

2. https://github.com/otaq/sortable-rails
Which is demoed here: http://rubaxa.github.io/Sortable/
It looks nice, but does not appear to be accessible.
